chainerx_cc/chainerx/native/native_device/linalg.cc

```diff
@@ -31,6 +32,38 @@ void sgetrf_(int* m, int* n, float* a, int* lda, int* ipiv, int* info);
 // getri
 void dgetri_(int* n, double* a, int* lda, int* ipiv, double* work, int* lwork, int* info);
 void sgetri_(int* n, float* a, int* lda, int* ipiv, float* work, int* lwork, int* info);
+
+// gesdd
+void dgesdd_(
+        char* jobz,
+        int* m,
+        int* n,
+        double* a,
+        int* lda,
+        double* s,
+        double* u,
+        int* ldu,
+        double* vt,
+        int* ldvt,
+        double* work,
+        int* lwork,
+        int* iwork,
+        int* info);
+void sgesdd_(
```

Comment:
A tiny but let's add empty lines between these function declarations (an empty line above this line).